neo/rawio/plexonrawio.py

```diff
@@ -209,6 +209,10 @@ def _parse_header(self):
                 for index, pos in enumerate(positions):
                     bl_header = data[pos : pos + 16].view(DataBlockHeader)[0]
 
+                    # To avoid overflow errors when doing arithmetic operations on numpy scalars
+                    np_scalar_to_python_scalar = lambda x: x.item() if isinstance(x, np.generic) else x
```

Comment:
Is there a reason not to make this lambda function near the top so it is reusable. Or even define this as a true private function. How are you thinking about using the same lambda function twice being generated in the for loop. It should be relatively low cost although seems a bit of a waste no? We could define the function once and reuse in both the for loops, no?

Reply:
Keeping the definition close to the use so people can quickly see what it is for. This takes nanoseconds:

![image](https://github.com/user-attachments/assets/4f76432c-cb14-455a-ac91-fc110a82ec90)

Extracting the data from plexon takes minutes. It does not factor.

Reply:
I figured you'd have checked. Fair enough. :)